resources/es/cemu.po

```diff
@@ -194,6 +310,180 @@ msgstr "Reiniciar"
+#: ChecksumTool.cpp
+msgid "Export successful"
+msgstr "Exportación correcta"
+
+#: ChecksumTool.cpp
+msgid "Can't write to file: {}"
+msgstr "No se puede escribir en el archivo: {}"
+
+#: ChecksumTool.cpp
+msgid "JSON file doesn't satisfy needed schema"
+msgstr "El archivo JSON no satisface el esquema necesario"
+
+#: ChecksumTool.cpp
+msgid "Wrong title id: {}"
+msgstr "ID de título incorrecto: {}"
+
+#: ChecksumTool.cpp
+msgid "Wrong version: {}"
+msgstr "Versión incorrecta: {}"
+
+#: ChecksumTool.cpp
+msgid "Wrong region: {}"
+msgstr "Región incorrecta: {}"
+
+#: ChecksumTool.cpp
+msgid "The verification data doesn't include a wud hash!"
+msgstr "¡Los datos de comprobación no incluyen un hash de WUD!"
+
+#: ChecksumTool.cpp
+msgid ""
+"Your game image is invalid!\n"
+"\n"
+"Your hash:\n"
+"{}\n"
+"\n"
+"Expected hash:\n"
+"{}"
+msgstr ""
+"¡Su imagen de juego no es válida!\n"
+"\n"
+"Su hash:\n"
+"{}\n"
+"\n"
+"Hash esperado:\n"
+"{}"
+
+#: ChecksumTool.cpp
+msgid "Select a file to export the errors"
+msgstr "Seleccione un archivo para exportar los errores"
+
+#: ChecksumTool.cpp
+msgid "Can't open file to write!"
+msgstr "¡No se puede abrir el archivo para escritura!"
+
+#: ChecksumTool.cpp
+msgid "The following files are missing:"
+msgstr "Faltan los siguientes archivos:"
+
+#: ChecksumTool.cpp
+msgid ""
+"{} files are missing!\n"
+"Do you want to export a list of them to a file?"
+msgstr ""
+"¡Faltan {} archivos!\n"
+"¿Desea exportar una lista de ellos a un archivo?"
+
+#: ChecksumTool.cpp
+msgid ""
+"{} files have an invalid hash!\n"
+"Do you want to export a list of them to a file?"
+msgstr ""
+"¡{} archivos tienen un hash no válido!\n"
+"¿Desea exportar una lista de ellos a un archivo?"
+
+#: ChecksumTool.cpp
+msgid ""
+"Multiple issues with your game files have been found!\n"
+"Do you want to export them to a file?"
+msgstr ""
+"¡Se han encontrado varios problemas con sus archivos de juego!\n"
+"¿Desea exportarlos a un archivo?"
+
+#: ChecksumTool.cpp
+msgid "Your game files are valid"
+msgstr "Sus archivos de juego son válidos"
+
+#: ChecksumTool.cpp
+msgid "JSON parse error: {}"
+msgstr "Error de análisis JSON: {}"
+
+#: ChecksumTool.cpp
+msgid "Can't open file!"
+msgstr "¡No se puede abrir el archivo!"
+
+#: ChecksumTool.cpp
+msgid "Can't parse json file!"
+msgstr "¡No se puede analizar el archivo JSON!"
+
+#: ChecksumTool.cpp
+msgid "Open checksum entry"
+msgstr "Abra una entrada de suma de comprobación"
+
+#: ChecksumTool.cpp
+msgid "Grabbing game files"
+msgstr "Recopilando archivos de juego"
+
+#: ChecksumTool.cpp
+msgid "Hashing game file: {}/{}"
+msgstr "Creando hash para archivo de juego: {} de {}"
+
+#: ChecksumTool.cpp
+msgid "Generated checksum of {} game files"
+msgstr "Suma de comprobación de {} archivos de juego generada"
+
+#: ChecksumTool.cpp
+msgid "Reading game image: {}"
+msgstr "Leyendo imagen de juego: {}"
+
+#: ChecksumTool.cpp
+msgid "Reading game image: {}/{}kb"
+msgstr "Leyendo imagen de juego: {} de {} KiB"
```

Comment:
Are you sure that it's "KiB" and not "KB"? Because they are actually different units.

Reply:
Sure, try to dump a game into the WUD format, and then get the file size (KiB is the correct term, MS Windows incorrectly uses "KB" when referring to that size, macOS or any "Linux-based operating system" has already fixed that issue).

Reply:
Ok! I don't have any WUD images at hand, so that's why I was asking